src/test/incremental/hashes/function_interfaces.rs

```diff
@@ -117,7 +117,7 @@ pub fn type_parameter() {}
 
 #[cfg(not(cfail1))]
 #[rustc_clean(cfg = "cfail2",
-              except = "Hir, HirBody, generics_of, type_of, predicates_of")]
+              except = "Hir, HirBody, generics_of, type_of, predicates_of,promoted_mir")]
```

Comment:
It's not clear to me why `promoted_mir` is dirty at any of the changes in this file.

Reply:
That's why we need to integrate the dependency tracking system into self-profiling `:)`